test/types/fetch.test-d.ts

```diff
@@ -1,7 +1,7 @@
 import { URL } from 'url'
 import { Blob } from 'buffer'
 import { expectType } from 'tsd'
-import { fetch, FormData, Headers, HeadersInit, Request, Response, RequestInit, ResponseInit } from '../..'
+import { BodyInit, fetch, FormData, Headers, HeadersInit, Request, RequestCache, RequestCredentials, RequestDestination, RequestInit, RequestMode, RequestRedirect, ResponseType, Response, ResponseInit } from '../..'
```

Comment:
Please make this multi-line :)

Reply:
`prettier` would've done it for me, if `undici` used it :wink: